stages/org.osbuild.ostree.genkey

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/python3
+"""Generate ed25519 public/private keypair in format used by `ostree sign`.
+
+This is used with the org.osbuild.ostree.sign stage, and these can be
+used with composefs to tie an initrd and ostree commit together. See
+https://ostreedev.github.io/ostree/composefs/#signatures for details
+of how this works.
+
```

Comment:
Let's add a note here that it "Requires 'openssl' in the buildroot." as documentation for using the stage.